zcash_client_sqlite/src/wallet.rs

```diff
@@ -273,32 +274,24 @@ pub fn get_balance_at<P>(
 /// use zcash_client_sqlite::{
 ///     NoteId,
 ///     WalletDB,
-///     wallet::get_received_memo_as_utf8,
+///     wallet::get_received_memo,
 /// };
 ///
 /// let data_file = NamedTempFile::new().unwrap();
 /// let db = WalletDB::for_path(data_file, Network::TestNetwork).unwrap();
-/// let memo = get_received_memo_as_utf8(&db, 27);
+/// let memo = get_received_memo(&db, 27);
 /// ```
-pub fn get_received_memo_as_utf8<P>(
-    wdb: &WalletDB<P>,
-    id_note: i64,
-) -> Result<Option<String>, SqliteClientError> {
-    let memo: Vec<_> = wdb.conn.query_row(
+pub fn get_received_memo<P>(wdb: &WalletDB<P>, id_note: i64) -> Result<Memo, SqliteClientError> {
```

Comment:
Now that this no longer returns just UTF-8 strings, the documentation needs updating.